src/lib/cli.ts

```diff
@@ -74,6 +71,11 @@ export class CliApplication extends Application {
                 if (this.json) {
                     this.generateJson(project, this.json);
                 }
+                if (!this.out && !this.json) {
+                    this.logger.log("You must either specify the 'out' or 'json' option.");
+                    this.logger.log("The './docs' directory have been set in default output");
```

Comment:
```suggestion
                    this.logger.log("The './docs' directory has be set as the output location by default");
```

Reply:
Of course, I correct that and repush
Sorry for the bad grammar :/
Thx for your suggestions =)